assets/js/search.js

```diff
@@ -206,6 +222,11 @@
             const searchResultClassName = 'top-banner-search--field-with-results--field--wrapper--search-component--search-results--result';
             if (!node || !_showingResults || node.classList.contains(CLASSNAME_HIGHLIGHTED)) return;
 
+            const link = node.querySelector('a');
```

Comment:
I think we should either move forward on this, or remove it and add it in a follow up PR....

Reply:
Do we want to log item hovers?

Reply:
Let's skip for now, for that exact data reason.